README.md

```diff
@@ -187,6 +187,14 @@ Following scripts helps with commonly used mass operations.
 
 - [bundle.sh](scripts/bundle.sh): Bundles all repositories which contains Gemfile
 - [update.sh](scripts/update.sh): Bundles repositories with Gemfile, runs npm build on UI, insights-proxy repositories and runs migrations on `topological_inventory-core` and `sources-api`
+- [haberdasher.sh](scripts/update.sh): Helps to run applications with using [haberdasher](https://github.com/RedHatInsights/haberdasher):
+  ```
+  Example 1 - with sending logs to kafka:
+  ./haberdasher.sh ./services/openshift-operations.sh kafka
+  
+  Example 2 - without prepeding haberdasher command:
+  ./haberdasher.sh ./services/openshift-operations.sh [kafka|stderr] without_hb
```

Comment:
[kafka|stderr] describes the parameter $2 as optional, but the script is always testing $3. 
Also $3 (without_db) there doesn't fit the check in the script (without_hb_cmd)

Reply:
yes it was confusing, so I improved validation and readme. thanks!